<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_view</class>
 <widget class="QWidget" name="main_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Microscope</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_login">
     <property name="title">
      <string>Login</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UserName</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_username">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_password">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_logout">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_tools">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Tools</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_open_camera">
        <property name="text">
         <string>Open Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_create_model">
        <property name="text">
         <string>Create Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_campare_images">
        <property name="text">
         <string>Compare Images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_detect_contours">
        <property name="text">
         <string>Detect Countors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
